### PR TITLE
Feat/block blog page

### LINKED ISSUE #68: Blog showing when typing in the url
Hi nice job! Just a question, if someone were to put in /blog after the URL, wouldn't the route still exist? We are only removing the button here. Maybe we can work on this enhancement later. A very minor issue tbh, great work on the commit 😄

_Originally posted by @Aryan3212 in https://github.com/chetanverma16/react-portfolio-template/issues/50#issuecomment-1180204152_

### PR DESCRIPTION
**Pull request by achu-krishna**
Added the feature to redirect from the blog page when url is entered to the previous page.

closes #68

### CODE REVIEW

**Review by chetanverma16 — CHANGES_REQUESTED** [review]
Awesome PR!
Everything looks good we can merge once the above is resolved

File: pages/blog/index.js [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
 const Blog = ({ posts }) => {
+  const showBlog = useRef(data.showBlog);
   const text = useRef();
   const router = useRouter();
   const [mounted, setMounted] = useState(false);
 
   useIsomorphicLayoutEffect(() => {
-    stagger([text.current], { y: 30 }, { y: 0 });
+    if (showBlog.current) stagger([text.current], { y: 30 }, { y: 0 });
+    else router.back();
```
[the remark this answers is not in the document]
Comment (chetanverma16) [reply pages/blog/index.js:18]: `router.push("/")`

**Comment by achu-krishna** [comment]
@chetanverma16 made the requested changes.

**Review by chetanverma16 — APPROVED** [review]
Awesome work!

**Pull request merged**